xml/SectorCategory.xml

```diff
@@ -458,7 +458,7 @@
                 <narrative xml:lang="fr"/>
             </description>
         </codelist-item>
-        <codelist-item status="withdrawn">
+        <codelist-item status="withdrawn" withdrawal-date="2020-02-28">
```

Comment:
I don’t think this `withdrawal-date` is correct. This code was withdrawn a long time ago – it wasn’t present in March 2017, so it was withdrawn earlier than that.

Reply:
@andylolz  I already emailed yesterday the OECD to check about all the withdrawn dates that were missing and ask them if they can specify the dates :)  I'll update here once I hear back. Thanks for cross-checking the date for this one.